src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/BankHolidayService.kt

```diff
@@ -1,34 +1,14 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
 
-import org.springframework.beans.factory.annotation.Value
 import org.springframework.cache.annotation.Cacheable
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.gov.GovUkApiClient
-import java.time.DayOfWeek
-import java.time.LocalDate
 
 @Service
 class BankHolidayService(
   private val govUkApiClient: GovUkApiClient,
-  @Value("\${maxNumberOfWorkingDaysAllowedForEarlyRelease}") private val maxNumberOfWorkingDaysAllowedForEarlyRelease: Int,
 ) {
 
   @Cacheable("bank-holidays")
   fun getBankHolidaysForEnglandAndWales() = govUkApiClient.getBankHolidaysForEnglandAndWales()
-
-  /** Friday is also considered as weekend */
-  fun isBankHolidayOrWeekend(releaseDate: LocalDate?): Boolean {
-    val listOfBankHolidays: List<LocalDate> = getBankHolidaysForEnglandAndWales()
-    val dayOfWeek = releaseDate?.dayOfWeek
```

Comment:
so the issue was because it was calling a method in the same class?

Reply:
yep that's right